src/inausoft.netCLI/OptionAttribute.cs

```diff
@@ -26,11 +31,28 @@ public class OptionAttribute : Attribute
         /// <summary>
         /// Initializes a new instance of <see cref="OptionAttribute"/> with specified name and description.
         /// </summary>
-        /// <param name="name"></param>
+        /// <param name="naming"></param>
         /// <param name="helpDescription"></param>
-        public OptionAttribute(string name, string helpDescription = "")
+        public OptionAttribute(string naming, string helpDescription = "")
         {
-            Name = name ?? throw new ArgumentNullException(nameof(name));
+            if (naming == null)
+            {
+                throw new ArgumentNullException(nameof(naming));
+            }
+
+            var names = naming.Split('|');
+
+            if(names.Length > 2 || names.Length == 0)
+            {
+                throw new ArgumentException(nameof(naming));
+            }
+
+            Name = names[0];
+
+            if(names.Length == 2)
+            {
+                ShortName = names[1];
```

Comment:
There is no limit for ShortName? It can be as long as we want?

Reply:
I guess no.. Should there be one?

Reply:
maybe not, leave as it is :)